main.lua

```diff
@@ -15,71 +15,134 @@ See the License for the specific language governing permissions and
 limitations under the License.
 
 --]]
-
-local bundle = require('luvi').bundle
+-- Convoluted boostrapping?
```

Comment:
Not sure what this comment adds.

Reply:
Looks like a leftover from when I was trying to sort out what the code does. I'll remove it!